lib/embulk/input/marketo/timeslice.rb

```diff
@@ -51,9 +101,16 @@ def transaction(config, &control)
               columns << Column.new(nil, name, type, column["format"])
             end
 
-            # TODO tasks should be executed concurrently.
-            resume(task, columns, 1, &control)
+            resume(task, columns, task[:workers], &control)
```

Comment:
As 3rd paramter, it should pass the total # of tasks to resume method. Please see https://github.com/embulk/embulk-input-s3/blob/master/embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java#L91